### DISCUSSION TITLE
Customize active dot background color

### DISCUSSION THREAD

**Discussion by jose-donato** [elrumordelaluz/reactour#534] [Q&A] [answered]
Hello. 

Amazing library!

Is it possible to customize active dot background color?

**Comment by elrumordelaluz** [comment, marked as answer]
Sure it is! Take a look into the ["custom styles" example](https://reactour.vercel.app/tour#custom-styles) and use the `dot` key into [styles prop](https://github.com/elrumordelaluz/reactour/tree/main/packages/tour#styles-stylesobj--popoverstylesobj--maskstylesobj-1): something like this:

```js
styles={{
 //  ...
 dot: (base, { current }) => ({
   ...base,
   background: current ? "#DAF7A6" : "#148F77"
 })
}}
```

Here is a [working example](https://codesandbox.io/s/objective-breeze-fq1s8m?file=/App.js:699-813).

**Reply by jose-donato** [reply]
Nice API 👍 

That's amazing, thanks.
